cloudbuild.yaml

```diff
@@ -14,6 +14,21 @@ steps:
     - --output=type=registry
     - --platform=linux/amd64,linux/arm64
     - .
+  # Build cloudbuild artifacts (for attestation)
+  - name: 'docker.io/library/golang:1.18.4-bullseye'
+    id: cloudbuild-artifacts
+    entrypoint: make
+    env:
+    # _GIT_TAG is not a valid semver, we use CI=1 instead
+    # - VERSION=$_GIT_TAG
+    - CI=$_CI
```

Comment:
So `$CI` is set by the prow job, but I'm not sure what is going on here.  Guessing this is an artifact from kops and can be removed?

Reply:
Yeah, this is a leftover from kOps. Think the makefile used this to determine how to create the build version. But the kOps Makefile does not contain any reference to this variable anymore. Can probably be removed.

Reply:
removed reference to _CI